Projects/UOContent/Multis/Houses/BaseHouse.cs

```diff
@@ -2408,7 +2423,17 @@ public void AddSecure(Mobile m, Item item)
                 }
                 else
                 {
-                    info = new SecureInfo((Container)item, SecureLevel.Owner);
+                    /* There seems to be some confusion regarding security access on secured containers. 
```

Comment:
The owner of the house should have access to everything.
This is both necessary and a problem since owners can hijack co-owner/friend property.
I am not sure this can be resolved.

Reply:
I agree. Owners will automatically have access to everyone. Perhaps we can make a new securelevel.HouseOwner?

Reply:
Maybe we need to separate the security access levels and the house access levels. These are obviously not interchangeable. The security access levels only apply to each individual container in question and are applicable to the individual who secured said container, NOT the house owner (although they have access/control over it regardless).

For things like doors and teleporters, the owner is automatically the house owner.